docs/conf.py

```diff
@@ -156,7 +161,20 @@ def setup(sphinx):
 # Add any extra paths that contain custom files (such as robots.txt or
 # .htaccess) here, relative to this directory. These files are copied
 # directly to the root of the documentation.
-html_extra_path = ["_static/css", "_static/robots.txt"]
+html_extra_path = ["_static/css"]
+
+# List of templates of static files to be included in the HTML output.
+# Keys represent paths to input files and values are dicts containing:
+# - target: The path where the rendered template should be placed.
+# - context: A dictionary listing variables that can be used inside the template.
+# All paths must be absolute.
+# Rendered templates are automatically added to html_extra_path setting.
+html_extra_templates = {
+    os.path.join(ROOT_PATH, "robots.txt.template"): {
+        'target': os.path.join(ROOT_PATH, "_static/robots.txt"),
+        'context': {'LATEST_VERSION': version},
```

Comment:
Was looking for this in the whole PR, and this is the key line for what the shell script did. Replacing `LATEST_VERSION` with version :)

Reply:
Yeah. The replacement could just be a one-liner. The hard part is just getting access to the necessary values (version, path to the project dir) and running it at the right stage (during the build rather than when someone just imports the config).